src/services/BlockRouter/BlockRouter.ts

```diff
@@ -15,15 +15,15 @@ interface EventMap {
   intersection: BlockIntersectionEvent;
 }
 
-const normalizeAnchor = (rawAnchor: string) => rawAnchor.replace('#', '');
+const normalizeAnchor = (rawAnchor: string) => rawAnchor?.replace('#', '');
```

Comment:
тогда, по всей видимости
```suggestion
const normalizeAnchor = (rawAnchor?: string) => rawAnchor?.replace('#', '');
```

Reply:
+